src/main.js

```diff
@@ -23,31 +24,36 @@ var load = loader({
   IndexedTask: {
     requires: ['cfg', 'monitor'],
     setup: ({cfg, monitor}) => data.IndexedTask.setup({
-      account:          cfg.app.azureAccount,
-      table:            cfg.app.indexedTaskTableName,
-      credentials:      cfg.taskcluster.credentials,
-      authBaseUrl:      cfg.taskcluster.authBaseUrl,
-      monitor:          monitor.prefix('table.indexedtasks'),
+      tableName:    cfg.app.indexedTaskTableName,
+      credentials:  sasCredentials({
+        accountId:  cfg.azure.accountId,
+        tableName:  cfg.app.indexedTaskTableName,
+        rootUrl:    cfg.taskcluster.rootUrl,
+        credentials:cfg.taskcluster.credentials,  
+      }),
     }),
   },
+
   Namespace: {
     requires: ['cfg', 'monitor'],
     setup: ({cfg, monitor}) => data.Namespace.setup({
```

Comment:
https://pastebin.mozilla.org/9087704
This is the only error I have left, because the AZURE_ACCOUNT env var hasn't been specified